wscript

```diff
@@ -312,7 +312,7 @@ def _build_dynamic_sources(bld):
     if bld.get_board().with_uavcan or bld.env.HAL_WITH_UAVCAN==True:
         bld(
             features='uavcangen',
-            source=bld.srcnode.ant_glob('modules/uavcan/dsdl/uavcan/**/*.uavcan'),
+            source=bld.srcnode.ant_glob('modules/uavcan/dsdl/* uavcan_vendor_specific_types/*', dir=True, src=False),
```

Comment:
could we put this under libraries/AP_UAVCAN/dsdl instead of in the root directory?
Also, won't this wildcard change mean we now match files that don't end in *.uavcan ? That could lead to parsing of leftover files from editing (eg. ~ files from emacs)

Reply:
Yes, we can put it under libraries/AP_UAVCAN/dsdl

No, this ant_glob returns only directories, not regular files (dir=True, src=False)